app/src/org/commcare/network/HttpUtils.java

```diff
@@ -1,9 +1,9 @@
 package org.commcare.network;
 
 import android.content.SharedPreferences;
-import android.support.v4.util.Pair;
 
 import org.commcare.CommCareApplication;
+import org.commcare.modern.util.Pair;
```

Comment:
Yes; I forget the exact chain of code that made this necessary, but I changed which type of `Pair` I was using somewhere else (because `android.support.v4.util.Pair` isn't accessible in `core`), and there was some reason that it was cleaner/easier to have the type here match.